File: nse/react/main src/Test2.js
import React, { Component } from 'react';
 
var p1=React.createElement('h1',null,"framework " )

class Test2 extends Component{

    render(){
        return(
            // <div>
            //  {p1}
            // </div>
            React.createElement('h1',this.props,"framework "+this.props.framework )
           )
    }
}
export default Test2

File: wipro/corejava/src/Test2.java
public class Test2 {
    public static void main(String[] args) {
        int x=10;
        x=x+1;
        System.out.println(x);
        x=x+2;
        System.out.println(x);
        //x=x+1 -- same x+=1
        //x=x+2 -- same x+=2
        x+=2;
        System.out.println(x);
    }
}
